src/core/connection/Generation.cpp

```diff
@@ -555,7 +557,46 @@ namespace Qv2ray::core::connection
                 //
                 root["api"] = GenerateAPIEntry(API_TAG_DEFAULT);
             }
+
+            if (!isComplex && GlobalConfig.inboundConfig.useTPROXY && GlobalConfig.outboundConfig.mark > 0)
```

Comment:
I suggest moving this check into the `simple connection config` if-branch, for simplicity.

Reply:
anyway, it just works.